src/codemagic/apple/app_store_connect/versioning/beta_build_localizations.py

```diff
@@ -0,0 +1,73 @@
+from dataclasses import dataclass
+from typing import List
+from typing import Optional
+from typing import Type
+from typing import Union
+
+from codemagic.apple.app_store_connect.resource_manager import ResourceManager
+from codemagic.apple.resources import Build
+from codemagic.apple.resources import ResourceId
+from codemagic.apple.resources import ResourceType
+from codemagic.apple.resources.beta_build_localization import BetaBuildLocalization
+from codemagic.apple.resources.enums import Locale
+
+
+class BetaBuildLocalizations(ResourceManager[BetaBuildLocalization]):
+    """
+    Beta Build Localizations
+    https://developer.apple.com/documentation/appstoreconnectapi/prerelease_versions_and_beta_testers/beta_build_localizations
+    """
+
+    @property
+    def resource_type(self) -> Type[BetaBuildLocalization]:
+        return BetaBuildLocalization
+
+    @dataclass
+    class Filter(ResourceManager.Filter):
+        build: Optional[ResourceId] = None
+        locale: Optional[Locale] = None
+
+    def create(self, build: Union[ResourceId, Build], locale: Locale, whats_new: str) -> BetaBuildLocalization:
```

Comment:
`whats_new` seems to be optional as per [docs](https://developer.apple.com/documentation/appstoreconnectapi/betabuildlocalizationcreaterequest/data/attributes)
```suggestion
    def create(self, build: Union[ResourceId, Build], locale: Locale, whats_new: Optional[str] = None) -> BetaBuildLocalization:
```

Of course then the method body should also be altered to include `whatsNew` attribute only in case it is given.